Formula/gimp.rb

```diff
@@ -1,87 +0,0 @@
-class Gimp < Formula
```

Comment:
Why do we want remove this formula? As far as I can tell, some users prefer to install software from homebrew even it's availabile from cask. The reasons can be preferring building from source, preferring homebrew's sandbox, preferring DIY build options, etc.